post_office/settings.py

```diff
@@ -79,6 +79,10 @@ def get_batch_size():
     return get_config().get('BATCH_SIZE', 100)
 
 
+def get_celery_enabled():
+    return get_config().get('CELERY', True)
```

Comment:
naming this configuration just `CELERY` is certainly a bad idea, since you just want to enable/disable it for `django-post_office`. If someone doesn't like Celery at all, those tasks are never triggered.

Reply:
![image](https://user-images.githubusercontent.com/24996016/93065654-46f29480-f679-11ea-8adc-186f468f4f71.png)

As the configuration is into the postoffice item, it wil not interfer with celery global configuration.
But I can rename it at "CELERY_TASKS_ENABLED"

Reply:
Yes, `CELERY_TASKS_ENABLED ` is more meaningful.